Formula/yaegi.rb

```diff
@@ -0,0 +1,23 @@
+class Yaegi < Formula
+  desc "Yet another elegant Go interpreter"
+  homepage "https://github.com/containous/yaegi"
+  url "https://github.com/containous/yaegi/archive/v0.7.2.tar.gz"
+  sha256 "1f2e63290614e26ffb82ef16af23d0ce5237c10a19a5c1191c22398adb62bc1e"
+  head "https://github.com/containous/yaegi.git"
+
+  depends_on "go" => :build
+
+  def install
+    dir = buildpath/"src/github.com/containous/yaegi"
+    dir.install buildpath.children
+
+    cd dir do
+      system "go", "build", "-o", bin/"yaegi", "cmd/yaegi/yaegi.go"
+      prefix.install_metafiles
+    end
+  end
```

Comment:
Since we replaced `GOPATH` we don't need to do `dir.install` part and so on.
I think the whole install method could look like:

```ruby
def install
  system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", bin/"yaegi", "cmd/yaegi/yaegi.go"
  prefix.install_metafiles
end
```

Reply:
Done and tested. It worked beautifully but I had to learn some stuff about squashing/fixing up :)